src/Raven.Server/Documents/Indexes/Index.cs

```diff
@@ -903,14 +903,27 @@ public virtual void Update(IndexDefinitionBase definition, IndexingConfiguration
 
                 _indexStorage.WriteDefinition(definition);
 
+                if (definition.LastStateChangeRaftIndex != Definition.LastStateChangeRaftIndex)
+                {
+                    if (definition.State == IndexState.Normal)
+                    {
+                        SetState(IndexState.Normal);
+                        stateChanged = true;
+                    }
+                    else
+                    {
+                        SetState(definition.State);
+                    }
+                }
```

Comment:
Can it be simplified to :
```
stateChanged = definition.State == IndexState.Normal;
SetState(definition.State);
```

Reply:
Does `stateChanged` means `EnableRequired` ?

Reply:
No